README.md

```diff
@@ -3,84 +3,137 @@
-V1-formatted files.
-
-Library documentation can be found [here](https://conda-incubator.github.io/conda-recipe-manager/index.html).
 
 ## History
 This project started out as a recipe parsing library in Anaconda's
-[percy](https://github.com/anaconda-distribution/percy) project.
+[percy](https://github.com/anaconda-distribution/percy) project. Some `git` history was lost during that transfer
+process.
+
+For those of you who come from `conda-forge`, you may associate CRM as "the tool that converts recipe files for
+`rattler-build`". Admittedly, that was the first-use case of the parsing capabilities provided by in this library. In
+the future, we aim to expand past that and offer a number of recipe automation tools and modules.
 
 # Getting Started
 
 ## General Installation
 
-### Install into your current environment
+To install the project to your current `conda` environment, run:
 ```sh
-make install
+conda install -c conda-forge conda-recipe-manager
 ```
+This will add the commands `conda-recipe-manager` and `crm` to your environment's path. Note that both of these
+commands are the same. `crm` is provided for convenience of typing.
+
+## CLI Usage
+Although CRM is a library, it does ship with a handful of command line tools. Running `crm --help` will provide a
+an up-to-date listing of all available tools. Run `crm <tool-name> --help` for usage documentation about each tool.
 
-### Install into a custom environment
+The following usage message was last updated on 2024-10-31:
 ```sh
-make environment
-conda activate conda-recipe-manager
+Usage: crm [OPTIONS] COMMAND [ARGS]...
+
+  Command line interface for conda recipe management commands.
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  convert             Converts a `meta.yaml` formatted-recipe file to the new
+                      `recipe.yaml` format.
+  graph               Interactive CLI for examining recipe dependency graphs.
+  patch               Modify recipe files with JSON patch blobs.
+  rattler-bulk-build  Given a directory, performs a bulk rattler-build
+                      operation. Assumes rattler-build is installed.
 ```
 
-# Developer Notes
+A high-level overview of the CLI tools can be found [here](./conda_recipe_manager/commands/README.md).
+
+## Developer Installation and Notes
+The `make dev` directive will configure a `conda` environment named `conda-recipe-manager` for you with
+a development version of the tooling installed.
+
 ```sh
 make dev
 conda activate conda-recipe-manager
 ```
-The `dev` recipe will configure a `conda` environment named `conda-recipe-manager` with
-development tools installed.
 
-`pre-commit` is automatically installed and configured for you to run a number
-of automated checks on each commit.
+### Developer Documentation
+We aim for a very high bar when it comes to code documentation so that we may leverage automatic documentation
```

Comment:
Why is this sentence required?

Reply:
It is attempt to "raise the bar" when it comes to our engineering excellence practices.

I'm putting this in the README with the hope that future contributors will commit documentation. We have a linter check for this but in my experience, it isn't perfect at catching missing docs.

Now I thought we had a contribution doc auto-imported by the conda org rules, but I'm not seeing it. I should probably add one as part of this PR.